### DISCUSSION TITLE
为什么bot不能正确匹配消息

### DISCUSSION THREAD

**Discussion by yllsyx** [Tsuk1ko/CQ-picfinder-robot#340] [问题求助] [answered]
![J~QZYJ11HPI_~@6Q RHC{ 3](https://user-images.githubusercontent.com/39957638/178154445-82208991-af14-4b7d-8bbb-668f769d8776.png)
我只修改了正则表达式里的名称

**Comment by Tsuk1ko** [comment, marked as answer]
你是在私聊吧，搜图模式是群聊功能

**Reply by yllsyx** [reply]
可是私聊也能触发搜图功能
![Screenshot_20220711_013434_com tencent mobileqq](https://user-images.githubusercontent.com/39957638/178155739-46d496f9-ceab-460d-8f28-843efb913fd0.jpg)
如果是群聊就是这种情况，很多次都是匹配不到，但是不知道为什么又能正确回应一句

**Reply by yllsyx** [reply]
搜图功能是能执行的，但是模式的开关之类的控制不了

**Reply by Miuzarte** [reply]
就是这么设计的  私聊直接发图没有开关

**Reply by Miuzarte** [reply]
还有  
![image](https://user-images.githubusercontent.com/66856838/178156248-6fd416da-8a89-4c9e-b916-f7bb9fe7e355.png)
不用带at

**Reply by Tsuk1ko** [reply]
https://github.com/Tsuk1ko/cq-picsearcher-bot/wiki/%E5%A6%82%E4%BD%95%E9%A3%9F%E7%94%A8#%E6%97%A5%E5%B8%B8%E9%A3%9F%E7%94%A8

**Reply by Tsuk1ko** [reply]
进出搜图模式不要带@

**Reply by yllsyx** [reply]
嗷，确实如此，是我猪鼻了o(╥﹏╥)o
